horovod/common/gradient_aggregation.py

```diff
@@ -1,6 +1,21 @@
 import tensorflow as tf
 
 
+def apply_op_to_not_none_tensors(tensor_op, tensors, *args):
+    resulting_tensors = []
+    for tensor in tensors:
+        if tensor is not None:
+            resulting_tensors.append(tensor_op(tensor, *args))
+        else:
+            resulting_tensors.append(tensor)
```

Comment:
thoughts on

```
return [tensor_op(tensor, *args) if tensor is not None else tensor for tensor in tensors]
```
?

it is more efficient

Reply:
Updated this.